src/derivatives/atlas.md

```diff
@@ -297,8 +293,8 @@ A guide for using macros can be found at
             "sub-01_atlas-AAL_dseg.nii.gz": "",
             "sub-01_atlas-AAL_dseg.tsv": "",
             "sub-01_atlas-AAL_probseg.nii.gz": "",
-            "sub-01_label-brain_mask.nii.gz": "",
-            "sub-01_label-head_mask.nii.gz": "",
+            "sub-01_atlas-AAL_seg-brain_mask.nii.gz": "",
+            "sub-01_atlas-AAL_seg-head_mask.nii.gz": "",
```

Comment:
Besides the label/seg discussion above. This example wants to exactly mean that the brain mask and the head mask WERE NOT projected from a template with those segments defined as part of the AAL atlas. Happy to add this clarification if necessary.

```suggestion
            "sub-01_label-brain_mask.nii.gz": "",
            "sub-01_label-head_mask.nii.gz": "",
```